src/Illuminate/Auth/AuthServiceProvider.php

```diff
@@ -48,7 +49,7 @@ protected function registerAuthenticator()
 	 */
 	protected function registerUserResolver()
 	{
-		$this->app->bind('Illuminate\Contracts\Auth\Authenticatable', function($app)
+		$this->app->bind(Authenticatable::class, function($app)
```

Comment:
Does `$app->make('Illuminate\Contracts\Auth\Authenticatable') === $app->make('\Illuminate\Contracts\Auth\Authenticatable')`? Saw some issue while back with some of this inconsistency (but not sure if it was ever fixed).